README.md

```diff
@@ -15,10 +15,19 @@ Here you will find all resources which will be given out throughout your time he
 * In order to have the demos from lectures you must checkout your cohorts branch ``` git checkout <month-day-year>```and run ```git pull```.
 * If there are changes done to assessment_tips, foundations_remote, student_success, or study_guides you must go into master and run ```git pull``` to receive the changes.
 
-# Student Section 
+# Student Section
 
-## Alumni Engagement Agreement
-https://appacademy.io/alumni-engagement-agreement
+## Instructor Contact Info 
+* All Instructors 
+    * instructors-sf@appacademy.io
+* Bay Bridge Instructors
+    * bay-bridge-instructors@appacademy.io
+* Golden Gate Instructors 
+    * golden-gate-instructors@appacademy.io
+
+## Assessment Policy
+* The App Academy Assessment Policy can be found [here](https://docs.google.com/document/d/1h_fKuhRYP5mGzTZrJHRyp9cpnBPlv5E7TjY_5pLZPNk/edit#).
```

Comment:
I think we should include something in the foundation's failure policy that includes that you can only defer once.